datastore-api/tests/test_kg_nodes.py

```diff
@@ -0,0 +1,64 @@
+import ipdb
+import json
+
+class TestKGNodes:
+
+    def test_get_nodes(self, client, kg_name, test_node, token):
+        # Given:
+        nid = test_node['id']
+        url = f"/datastores/kg/{kg_name}/{nid}"
+
+        # When:
+        response = client.get(url, headers={"Authorization": f"Bearer {token}"})
```

Comment:
I think we do not need the token here, since it is mocked in the contest (i.e. `app.dependency_overrides[client_credentials] = lambda: token`)